### DISCUSSION TITLE
containerOverrides deprecated?

### DISCUSSION THREAD

**Discussion by biofilos** [aws/amazon-genomics-cli#548] [Q&A]
Hi, I am trying to get agc working in our environment.

When I sent a job, I saw the following message in Batch

```
Configuration conflict
This value was submitted using containerOverrides.vcpus which has been deprecated and was not used as an override. Instead, the VCPU value found in the job definition’s resourceRequirements key was used instead. More information about the deprecated key can be found in the AWS Batch API documentation. 
```

Is this message of concern?. A similar message appears in the memory allocation section.  
I saw it on a task that failed with out-of-memory error, even though the task takes way less memory than what I provided

**Comment by wleepang** [comment]
Which `engine` are you using for your workflows? This is a result of an engine not using the new request shape that places vCPU and Memory specifications in the `resourceRequirements` property when it it submits AWS Batch Jobs.

**Comment by markjschreiber** [comment]
I believe this issue occurs with Cromwell because AGC is using an older version which uses the deprecated method of creating AWS Batch Tasks. We are looking at updating in the near future. If the engine is not Cromwell please let us know.

In the meantime, I don't think that using the deprecated API will give an misconfigured container.
